src/layouts/BlogLayout.astro

```diff
@@ -132,4 +135,57 @@ article :global(figcaption) {
 	font-size: var(--step--1);
 	font-style: italic;
 }
+
+article :global(table) {
```

Comment:
This `:global()` is needed because we are trying to style the content inside the `<slot>`

Reply:
Ahh, thank you